templates/create_buckets.mustache

```diff
@@ -0,0 +1 @@
+The following buckets were created: {{buckets}}
```

Comment:
In `cogctl` I've been formatting things like the following, which looks pretty good if you want to try it out here instead of using a comma separated list.

```
Created bucket-test1
Created bucket-test2
```